sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/CSharpLanguageConfiguration.java

```diff
@@ -26,4 +26,8 @@ public class CSharpLanguageConfiguration extends AbstractLanguageConfiguration {
   public CSharpLanguageConfiguration(Configuration configuration) {
     super(configuration, CSharpPlugin.LANGUAGE_KEY);
   }
+
+  public boolean analyzeRazorCode() {
```

Comment:
Do we need also logic on the plugin side? Even if we don't report from the analyzers?

Reply:
We don't use it. I added it since that way I could test that the default value of the property is indeed true, and I would say it does not do any harm there. We might even consider adding a debug log so if users turn it off and we need to troubleshoot their razor analysis  we could check it easily what the parameter was.